### PR TITLE
Clinical presentation slides complete (mostly)

### PR DESCRIPTION
**Pull request by robinandeer**
I'll probably push changes to the text content here laters!

### CODE REVIEW

**Comment by robinandeer** [comment]
I'm ready for a really brutal "code" review ^^

(sorry about the file size...)

File: parts/3-clinical/slides.md [hunk 1]
```diff
@@ -1,124 +1,113 @@
...
+The kinds of disoders we are talking about here are rare and inherited from parent to
+child. The cause is, for the sake of this presentation 100% genetic and often develop
+early in life of the patients and cause severe symptoms.
 
-## Why?
-Patients can go undiagnosed for 15 years
-Todays blunt tests give only vague indications of waht could be cusing these
-disoders.
+Right now the process is quite bleak. Patients meet with a doctor, run some tests,
+are sent home, some more tests are run, the see another doctor, home again... This
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: they/then

File: parts/3-clinical/slides.md [hunk 2]
```diff
@@ -1,124 +1,113 @@
...
+facto delivery report that doctors look at after they've sent in samples for sequencing.
+
+### Tech stack
+We're using a MongoDB database, I'll get back to this later, with a Flask server.
+Basically because this is what I was familiar with. We also make extensive use of
+other thrid party libraries such as Jinja2 for HTML templates and MongoEngine as an
+API for our database.
+
+We have to remember that we are dealing with sensitive clinical data. We are by no
+means security experts but what we do, do is to restrict access based on IP adresses,
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: -do :)

File: parts/3-clinical/slides.md [hunk 3]
```diff
@@ -1,124 +1,113 @@
...
-can't make use of aggresive normalization even if we wanted. This makes MongoDB
-attractive as well!
-
-### MongoEngine ODM as the API
-MongoEngine as an interface to MongoDB - we've built our API around it!
-Finally a chance to show some code examples ^^
+There are a few undeniable upsides such as the much simplified and more intuitive data model with e.g. nested comments.
+
+Most importantly: if a doctors shows up and wants to take a look at a result from way
+back we need to be able to show her the exactly what the previous clinical
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: -the :-P

File: parts/3-clinical/slides.md [hunk 4]
```diff
@@ -1,124 +1,113 @@
...
-- Just to give an insight into my process I could explain how they
-  handle a simple logout form. Under the button hides a sign out form
-  that gets submitted.
+I have two take-away messages to share. The first is how important it's been for us
+to be able to come together under a single language to do just about everthing we
+need to run a sequencing platform. Python allows us to do that unlike any other
+language.
+
+The second message is what a huge potential we have in science/academia of being
+leveraging open source development. Bioinformatics has a hig and lively open source
```
Comment (guillermo-carrasco) [inline parts/3-clinical/slides.md]: high? big?

**Comment by guillermo-carrasco** [comment]
I've finished my *brutal* "code" review :)

Nice additions! I really like your slides, makes me feel bad... hehe no, I'm kidding. 

I like the memes added to the intro, just let's see how I manage to fit all I have to say in 12.5 mins

**Comment by robinandeer** [comment]
Thank you!

I feel like it's difficult to help you in isolation with the intro-slides but I do my best :smiley:

**Pull request merged**